spec/models/membership_spec.rb

```diff
@@ -132,12 +138,26 @@
     end
   end
 
-  context 'locked group' do
+  context 'locked membership' do
     it 'cannot be destroyed' do
+      group = FactoryGirl.create(:finalizing_group)
+      membership = group.memberships.first
+      expect { membership.destroy }.not_to change { group.memberships_count }
+    end
+    it 'must be accepted' do # rubocop:disable RSpec/ExampleLength
+      group = FactoryGirl.create(:finalizing_group)
+      membership = Membership.new(group: group,
+                                  user: FactoryGirl.create(:student,
```

Comment:
very very minor thing, but just to be consistent, I would probably define user separately like group was defined. Then the line spacing isn't as awkward